services/helper/sessions.js

```diff
@@ -1,64 +1,703 @@
+                    return common.failureResponse({
+                        message: apiResponses.NOT_A_MENTOR,
+                        statusCode: httpStatusCode.bad_request,
+                        responseCode: 'CLIENT_ERROR'
+                    });
+                }
+
+                const session = await sessionData.findSessionById(sessionId);
+
+
+                if (!session) {
+                    return resolve(common.failureResponse({
+                        message: apiResponses.SESSION_NOT_FOUND,
+                        statusCode: httpStatusCode.bad_request,
+                        responseCode: 'CLIENT_ERROR'
+                    }));
+                }
+
+                if (session.userId !== mentor.data.result._id) {
+                    return resolve(common.failureResponse({
+                        message: apiResponses.CANNOT_START_OTHER_MENTOR_SESSION,
+                        statusCode: httpStatusCode.bad_request,
+                        responseCode: 'CLIENT_ERROR'
+                    }));
+                }
+
+                let link = "";
+                session.link = "";
+                if (session.link) {
+                    link = session.link;
+                } else {
+
+                    let currentDate = moment().utc().format(common.UTC_DATE_TIME_FORMAT);
+                    let elapsedMinutes = moment(session.startDateUtc).diff(currentDate, 'minutes');
+
+                    if (elapsedMinutes > 10) {
+                        return resolve(common.failureResponse({
+                            message: apiResponses.SESSION_ESTIMATED_TIME,
+                            statusCode: httpStatusCode.bad_request,
+                            responseCode: 'CLIENT_ERROR'
+                        }));
+                    }
+
+                    const meetingDetails = await bigBlueButton.createMeeting(
+                        session._id,
+                        session.title,
+                        session.menteePassword,
+                        session.mentorPassword
+                    );
+
+                    if (!meetingDetails) {
+                        return resolve(common.failureResponse({
+                            message: apiResponses.MEETING_NOT_CREATED,
+                            statusCode: httpStatusCode.internal_server_error,
+                            responseCode: 'SERVER_ERROR'
+                        }));
+                    }
+
+                    const moderatorMeetingLink = await bigBlueButton.joinMeetingAsModerator(
+                        session._id,
+                        mentorDetails.name,
+                        session.mentorPassword
+                    );
+
+                    await sessionData.updateOneSession({
+                        _id: session._id
+                    }, {
+                        link: moderatorMeetingLink,
+                        status: "live",
+                        startedAt: new Date()
```

Comment:
Lets save startedAt in utc format @aman-tunerlabs

Reply:
@aman-tunerlabs status can be taken from common.js